include/swift/Runtime/Config.h

```diff
@@ -216,6 +216,7 @@
 // But some of the runtime entries are invoked directly from
 // the foundation. Therefore they should be visible.
 #define SWIFT_RT_ENTRY_VISIBILITY SWIFT_RUNTIME_EXPORT
+#define SWIFT_EXTERN_RT_ENTRY_VISIBILITY SWIFT_EXTERN_RUNTIME_EXPORT
```

Comment:
I see, cheers. I assume this is because `extern "C"` means that the symbol name stays the same as the name of the method - i.e. not mangled. Do you think we need to make this a bit more explicit and self documenting, via some macro (`#define SWIFT_NON_MANGLED_SYMBOL extern "C"`) or something like that? 
Might not be worth it as probably most people digging into the runtime (apart from me) understand C linkages well. On the other hand, would make out intentions explicit